src/frontend/komponenter/Fagsak/FagsakContainer.tsx

```diff
@@ -9,27 +10,25 @@ import { DokumentutsendingProvider } from '../../context/DokumentutsendingContex
 import { useFagsakContext } from '../../context/fagsak/FagsakContext';
 import useSakOgBehandlingParams from '../../hooks/useSakOgBehandlingParams';
 import { useAmplitude } from '../../utils/amplitude';
-import Venstremeny from '../Felleskomponenter/Venstremeny/Venstremeny';
 import BehandlingContainer from './BehandlingContainer';
 import Dokumentutsending from './Dokumentutsending/Dokumentutsending';
-import Høyremeny from './Høyremeny/Høyremeny';
 import JournalpostListe from './journalposter/JournalpostListe';
 import Personlinje from './Personlinje/Personlinje';
 import Saksoversikt from './Saksoversikt/Saksoversikt';
 
+const Hovedinnhold = styled.div`
+    height: calc(100vh - 6rem);
+    overflow: auto;
+`;
+
 const FagsakContainer: React.FunctionComponent = () => {
     const { fagsakId } = useSakOgBehandlingParams();
 
     const location = useLocation();
     const { loggSidevisning } = useAmplitude();
     const erPåSaksoversikt = location.pathname.includes('saksoversikt');
-    const erPåDokumentliste = location.pathname.includes('dokumenter');
     const erPåDokumentutsending = location.pathname.includes('dokumentutsending');
 
-    const skalHaVenstremeny = !erPåSaksoversikt && !erPåDokumentliste && !erPåDokumentutsending;
-
-    const skalHaHøyremeny = !erPåSaksoversikt && !erPåDokumentutsending;
-
```

Comment:
denne logikken gjør to ting:
- venstremenyen rendres kun for BehandlingContainer
- høyremenyen rendres for BehandlingContainer + `/dokumenter`, men viser ingenting inne på dokumentlisten fordi den ikke finner en `åpenBehandling`

Før:
![image](https://user-images.githubusercontent.com/2379098/221563779-cbb13f45-9ba5-47e4-8951-463036e5f99b.png)